spec/rubocop/server/rubocop_server_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+RSpec.describe 'rubocop --server', :isolated_environment do # rubocop:disable RSpec/DescribeClass
+  let(:rubocop) { "#{RuboCop::ConfigLoader::RUBOCOP_HOME}/exe/rubocop" }
+
+  include_context 'cli spec behavior'
+
+  if RuboCop::Server.support_server?
+    context 'when using `--server` option after updating RuboCop' do
```

Comment:
And we should probably mention this in the docs as well.

Reply:
I've mentioned it in the doc.